### PR TITLE
fix the script and enhance the doc

### PR DESCRIPTION
**Pull request by brightzheng100**
two small updates:

1. fixed the script as in `v0.6.0`, `kind get nodes` will get nothing, so `--name` must be provided:

```
$ kind version
kind v0.6.0 go1.13.4 darwin/amd64
$ kind get nodes
$ kind get nodes --name my-cluster
my-cluster-worker3
my-cluster-worker2
my-cluster-control-plane
my-cluster-worker
```

2. enhanced a bit on the doc on how to use the registry.

### CODE REVIEW

File: site/content/docs/user/local-registry.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ with it enabled.
 set -o errexit
 
 # create registry container unless it already exists 
+CLUSTER_NAME="my-cluster"   # desired cluster name; default is "kind"
```
Comment (BenTheElder) [inline site/content/docs/user/local-registry.md]: maybe just make this `kind`?

Comment (BenTheElder) [reply site/content/docs/user/local-registry.md]: Can punt to a follow up

**Comment by BenTheElder** [comment]
/lgtm
/approve
Thanks!

**Comment by BenTheElder** [comment]
🙄 flakes

File: site/content/docs/user/local-registry.md (hunk 1 above)
Comment (brightzheng100) [reply site/content/docs/user/local-registry.md]: Sorry you're much faster than I could respond:)
I think having a dummy name like that might draw the right attention that "it's your decision" -- especially with a note there; while naming it `kind` may cause someone think of "is that a keyword"...

Anyway, thanks for merging it!

**Pull request merged**